deploy/crds/winoperators.config.openshift.io_windowsoperatorconfigs_crd.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: windowsoperatorconfigs.winoperators.config.openshift.io
+spec:
+  group: winoperators.config.openshift.io
```

Comment:
Are we sure we can use `config.openshift.io` for operators that perform day 2 operations? @sdodson @crawford please weigh in here.

Reply:
This has been resolved over slack. As per @deads2k, `config.openshift.io` must lifecycle on an OpenShift release cadence. and that the configuration should live with the component.